### DISCUSSION TITLE
Background Sounds not playing

### DISCUSSION THREAD

**Discussion by PianoMan7** [ExistentialAudio/BlackHole#787] [Q&A]
Henlo! 

I am trying to playback MacOS's built-in [background sounds](https://support.apple.com/en-ca/guide/mac-help/mchl3061cdc6/mac) (Sonoma 14.5) through a multi-output device, but I am unable to hear anything. 

Playback works with YouTube and all other apps that I have in the <1 week that I have had BlackHole. 

The playback of background sounds works if I set the output to be my MBP's speakers or my Steinberg UR22.

TIA!

**Comment by devinroth** [comment]
Odd. Might be some sort of macOS limitation. Again Multi-outputs are Apple’s software so unfortunately can’t do anything to support them.

Devin
